### ISSUE TITLE
Installation docs are out of date: gcs release bucket is stale

### ISSUE THREAD

**Issue by farcaller** [containerd/cri#934]
https://github.com/containerd/cri/blob/master/docs/installation.md mentions `storage.googleapis.com/cri-containerd-release` as the source of truth for tarballs and checksums but the last release out there is 1.0.2. The recent gihtub releases have no published signature.

**Comment by estesp** [comment]
I'm pretty sure the Google storage bucket is quite up to date; this was just published there last week after we released the latest 1.1.x patch release:
```
<Key>cri-containerd-1.1.4.linux-amd64.tar.gz</Key>
<LastModified>2018-09-27T18:15:48.530Z</LastModified>
```

Also, all the of the 1.2.0 betas and RC are also published there. What are you seeing?

**Comment by farcaller** [comment]
I was looking for 1.11.1 (https://github.com/containerd/cri/releases) but apparently what I was actually looking for was containterd itself, not cri.
